README.md

```diff
@@ -53,12 +53,6 @@ conda activate fgpyo
 poetry install
 ```
 
-If, during `poetry install` on Mac OS X errors are encountered running gcc/clang to build `pybedtools` or other packages with native code, try setting the following and re-running `poetry install`:
-```bash
-export CFLAGS="-stdlib=libc++"
-``` 
-
-
```

Comment:
why is this hint no longer needed?

Reply:
Because pybedtools is no longer a [transitive] dependency!